server/src/instant/dash/routes.clj

```diff
@@ -1008,10 +1009,18 @@
 ;; ---
 ;; CLI
 
+(defn- remove-system-namespaces [entities]
+  (ucoll/filter-keys
+   #(not (string/starts-with? (name %) "$"))
```

Comment:
Nit: I think you can use all-etypes from system-catalog.clj

Reply:
It feels like we should prevent modifying any namespaces that start with `$`, no? So that user won’t accidentally create `$folder` or smth that we later won’t be able to use

Reply:
Fair point! Down